lib/semian/protected_resource.rb

```diff
@@ -2,13 +2,14 @@ module Semian
   class ProtectedResource
     extend Forwardable
 
-    def_delegators :@bulkhead, :destroy, :count, :semid, :tickets, :registered_workers, :name
```

Comment:
If it's not too much trouble to create a failing test case for this, that'd be great